.lintstagedrc.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   '**/*.js': (files) => [
     `eslint --quiet --fix ${files.join(' ')}`,
-    `jest --passWithNoTests`,
+    `vitest --passWithNoTests`,
```

Comment:
`vitest` runs in watch mode by default outside CI environments. For single run, use [vitest run](https://vitest.dev/guide/cli.html#vitest-run)
```suggestion
    `vitest run --passWithNoTests`,
```